mainui/menu_newgame.cpp

```diff
@@ -71,6 +71,7 @@ UI_NewGame_StartGame
 */
 static void UI_NewGame_StartGame( float skill )
 {
+printf("StartGame(%f)\n", skill);

```

Comment:
Oh... just stop using printf(). 
And game already prints game skill when dev_level equal or greater than 1.